elements/pl-file-editor/pl-file-editor.js

```diff
@@ -100,6 +108,56 @@ window.PLFileEditor.prototype.updatePreview = function (html_contents) {
   }
 };
 
+window.PLFileEditor.prototype.initSettingsButton = function (uuid) {
+  var that = this;
+
+  this.settingsButton.click(function () {
+    ace.require(['ace/ext/themelist'], function (themeList) {
+      var themeSelect = that.modal.find('#' + uuid + '-themes');
+      themeSelect.empty();
+      for (const entries in themeList.themesByName) {
+        var caption = themeList.themesByName[entries].caption;
+        var theme = themeList.themesByName[entries].theme;
+
+        if (localStorage.getItem('pl-file-editor-theme') === theme) {
+          themeSelect.append(
+            $('<option>', {
+              value: theme,
+              text: caption,
+              selected: true,
+            })
+          );
+        } else {
+          themeSelect.append(
+            $('<option>', {
+              value: theme,
+              text: caption,
+            })
+          );
+        }
```

Comment:
I think you can refactor to this:

```suggestion
        themeSelect.append(
          $('<option>', {
            value: theme,
            text: caption,
            selected: localStorage.getItem('pl-file-editor-theme') === theme,
          })
        );
```